src/DnsStack.ts

```diff
@@ -66,6 +68,24 @@ export class DnsStack extends cdk.Stack {
       parameterName: Statics.envRootHostedZoneName,
     });
 
+    new CrossRegionParameter(this, 'csp-sub-hostedzone-id-us', {
+      region: 'us-east-1',
+      name: Statics.envRootHostedZoneId,
+      value: subzone.hostedZoneId,
+      description: 'Account root hosted zone ID',
+    });
+    new CrossRegionParameter(this, 'csp-sub-hostedzone-name-us', {
+      region: 'us-east-1',
+      name: Statics.envRootHostedZoneName,
+      value: subzone.zoneName,
+      description: 'Account root hosted zone name',
+    });
+
+    NagSuppressions.addStackSuppressions(this, [{
```

Comment:
The non-container Lambda function is not configured to use the latest runtime version. 

Use the latest available runtime for the targeted language to avoid  technical debt. Runtimes specific to a language or framework version are  deprecated when the version reaches end of life. This rule only applies  to non-container Lambda functions.